src/web_app_handler.py

```diff
@@ -33,21 +35,26 @@ def handle_event(self, data, headers):
             if self.dry_run:
                 return reason, http_code
 
+            # e.g. "https://api.github.com/repos/getsentry/sentry/actions/workflows/1174556",
+            org = data["workflow_job"]["url"].split("repos/")[1].split("/")[0]
```

Comment:
hmm there should be a property inside the `data['repository']` that contains `owner` rather than parsing it out of the url

Reply:
This would be in the case if we listened to workflow runs rather than workflow jobs. I don't think I will be allocated time to spend on this project unless we get external customers' interest.

Reply:
orite